src/core/annotation.js

```diff
@@ -1003,6 +1039,13 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
 
     this.data.exportValue =
       exportValues[0] === "Off" ? exportValues[1] : exportValues[0];
+
+    const normalAppearanceState = customAppearance.get("N");
```

Comment:
The spec seems to call this just `normalAppearance`, so let's use that name too.